js/src/dom/manipulator.js

```diff
@@ -44,7 +52,7 @@ const Manipulator = {
     }
 
     const attributes = {}
-    const bsKeys = Object.keys(element.dataset).filter(key => key.startsWith('bs'))
+    const bsKeys = Object.keys(element.dataset).filter(key => key.startsWith('bs') && !key.startsWith('bsConfig'))
 
     for (const key of bsKeys) {
       let pureKey = key.replace(/^bs/, '')
```

Comment:
@GeoSot shouldn't this remove `bsConfig` too?

Reply:
yes, I want to filter `bsConfig` out of given results . My purpose is to can be accessed only on purpose and not by massive get

Reply:
I'm not sure I understand. Right now, `bsConfig` is not normalized, right? Shouldn't it be normalized here?

Reply:
here (`getDataAttributes`) is omitted only on purpose.
 We may need to think of normalization only in config.js usage I suppose